File: Python/백준/Bronze/28453. Previous Level/Previous Level.py
a=int(input()); aa = list(map(int,input().split())); bb=[]
for i in range(a):
    if aa[i]<250:
        bb.append(4)
    elif aa[i]<275:
        bb.append(3)
    elif aa[i]<300:
        bb.append(2)
    else:
        bb.append(1)
print(*bb)

File: C#/백준/Bronze/28453. Previous Level/Previous Level.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            string[] aaa = Console.ReadLine().Split();
            int[] aa = Array.ConvertAll(aaa,int.Parse);
            for(int i=0; i<a; i++){
                if(aa[i]<250){
                    Console.Write("4 ");
                }
                else{
                    Console.Write($"{13-aa[i]/25} ");
                }
            }
        }
    }
}